src/guc.c

```diff
@@ -660,6 +662,17 @@ _guc_init(void)
 							 NULL,
 							 NULL);
 
+	DefineCustomBoolVariable(MAKE_EXTOPTION("enable_tss_callbacks"),
+							 "Enable ts_stat_statements callbacks",
+							 "Enable ts_stat_statements callbacks",
+							 &ts_guc_enable_tss_callbacks,
+							 false,
+							 PGC_USERSET,
```

Comment:
Do we want the user to be able to enable/disable this? Maybe this GUC should be restricted to super-user only.

Reply:
I think at least on cloud we don't want users disabling this.

Reply:
Good catch... also today during the Data Life Cycle standup we talked about making it ON by default

Reply:
Done